### ISSUE TITLE
[500]: \"\n\"guild_id\

### ISSUE THREAD

**Issue by Drxckzyz** [FerrisChat/Server#38]
Bug Report:
Api returns an Internal Server Error When Deleting Channel

Steps to Reproduce:
1. Create Channel
2. Delete created Channel

Code:
Link: https://github.com/FerrisChat/Ferris-lib/blob/main/test/test.js
```ts
const Ferrislib = require('../build/index')
const token = "MTA1NDc4Njk1NDUzODU2MjE4NzY3ODY5NDc2ODY0MA==.BVwW6vC4gw-B6MVJGqqaaGWIw1saNFvEOH7cSLbcmJb53lv-wv4mUHKlpy_c0PiJeXbR57dqMy77iv8a5VZd1g=="
const client = new Ferrislib.Client({
    shardCount: 1,
})

client.on("rawWs", console.log)
client.on("debug", console.log)
client.on("ready", async () => {
    await client.user.fetch()
    console.log(client.guilds)
    const g = await client.createGuild({ name: "test" })
    console.log(g)
    const c = await client.createChannel(g.id, { name: "test" })
    console.log(c)
    console.log(client.guilds.get(g.id))
    await client.deleteChannel(c.id)
    await client.deleteGuild(g.id)
    console.log(client.guilds)
})

client.login({ email: "=", password: "=" })
```

**Comment by tazz4843** [comment]
Should be a easy 4 line fix, just need to drop lines 12, 14, and 19 in delete_channel.rs, and update 17 to remove the now removed query arg. If you want to make a PR for it go ahead, otherwise I'll work on this tomorrow since it's a quick fix.

**Comment by Drxckzyz** [comment]
Alright i will create a PR to fix it

**Comment by tazz4843** [comment]
closed by #41
